src/js/medium-editor.js

```diff
@@ -1,4 +1,5 @@
 /*global module, console, define*/
+var DEBOUNCE_INTERVAL = 50;
```

Comment:
I had placed it there originally as I agree that's the right place for it. However, jslint was not a fan of a var being declared there.  Since this constant is really only used inside of `debounce()`, do you think we should just move its declaration in there?